### PR TITLE
task3

### PR DESCRIPTION
**Pull request by Samuelcrudge**
working foreach and print out of data.

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,32 @@
+<?php 
+require "php/functions.php";
+
+
+//$query = $database->prepare("SELECT * FROM `bread` ORDER BY `id` ASC;");
+//$query->execute();
```
Comment (bradleypurdue) [inline index.php]: Does this need to be here? It's generally a bad idea to commit commented code.

File: index.php [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>The Bread</title>
+<link rel="stylesheet" type="text/css" href="css/breadstyle.css">
+</head>
+<body>
+<nav>
+    <img src="">
```
Comment (bradleypurdue) [inline index.php]: Random `` tag.

File: index.php [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+
+
+//$query = $database->prepare("SELECT * FROM `bread` ORDER BY `id` ASC;");
+//$query->execute();
+?>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<title>The Bread</title>
+<link rel="stylesheet" type="text/css" href="css/breadstyle.css">
```
Comment (bradleypurdue) [inline index.php]: Indentation is off here.

File: index.php [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+        <li>
+            more bread
+        </li>
+    </ul>
+</nav>
+<div>
+    <?php
+    loopbread($breads);
+    ?>
+</div>
```
Comment (bradleypurdue) [inline index.php]: Indentation is off here.

File: php/functions.php [hunk 5]
```diff
@@ -0,0 +1,21 @@
...
+$db->setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+
+$query = $db->query("SELECT `name`, `type`, `rating`, `desc` FROM `bread`;");
+$breads = $query->fetchall();
+
+function loopbread($breads) {
+    foreach ($breads as $bread){
+        echo $bread['name'];
+
+        echo "<div>";
```
Comment (danicatt) [inline php/functions.php]: Question - by putting a div round each entry ,will this help you separate each one when you come to style/format it in CSS? I may have to pinch this approach!

Comment (Samuelcrudge) [reply php/functions.php]: yes makes flexing items into line easier :)

File: index.php (hunk 2 above)
Comment (Samuelcrudge) [reply index.php]: for structure in next task

**Pull request merged**